assets/img/sponsors/baroodycamps.svg

```diff
@@ -0,0 +1,24 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 281 297">
```

Comment:
@ErikBoesen you said that you had made changes to the file before. Where are those changes?

Reply:
All the other SVG logos.